hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -515,6 +515,12 @@ public function upgradeBundle() {
     $this->upgrade_1034();
     $this->upgrade_1035();
     $this->upgrade_1036();
+    $this->upgrade_1037();
```

Comment:
The call to the upgrader were missing from  previous PR for upgrade `1037 - 1041`.

Reply:
@ajesamson how come some many upgraders were missing? Are we not testing this changes in new sites?

Reply:
They are new upgraders recently added but was omitted in the previous PRs.

Reply:
@ajesamson but why were they omitted? The only reason I can think of is that this is not being properly tested by devs locally

Reply:
They were mistakenly omitted. Test was properly done locally as upgrades were carried out successfully and log also indicating their upgrade number without having to do call to them from the installer. After reviewing the page implementation again, it was then discovered they should have been triggered by the installer.